packages/core/src/index.js

```diff
@@ -24,12 +24,15 @@ export { FileListPlaceholder } from './FileListPlaceholder/FileListPlaceholder.j
 export { Help } from './Help/Help.js'
 export { Input } from './Input/Input.js'
 export { Label } from './Label/Label.js'
+export { Layer, useLayerContext } from './Layer/Layer.js'
 export { Legend } from './Legend/Legend.js'
 export { LinearLoader } from './LinearLoader/LinearLoader.js'
 export { Logo, LogoIcon, LogoIconWhite, LogoWhite } from './Logo/Logo.js'
 export { Menu } from './Menu/Menu.js'
+export { MenuDivider } from './MenuDivider/MenuDivider.js'
 export { MenuItem } from './MenuItem/MenuItem.js'
 export { MenuList } from './MenuList/MenuList.js'
```

Comment:
Do we need to export this one?

Reply:
Just checked with @cooper-joe, yes we do.